tests/test_types.py

```diff
@@ -1170,3 +1171,42 @@ class Model(BaseModel):
             'type': 'type_error.integer',
         },
     ]
+
+
+def test_valid_simple_json():
+    class JsonModel(BaseModel):
+        json_obj: Json
+
+    obj = {'a': 1, 'b': [2, 3]}
+    assert JsonModel(json_obj=json.dumps(obj)).dict() == {'json_obj': obj}
+
+
+class JsonDetailedModel(BaseModel):
```

Comment:
can you move the class definition into each test since all the field setup is done here and can fail.